### PR TITLE
AH-64D migrated to Module

### PR DESCRIPTION
**Pull request by jdahlblom**

### CODE REVIEW

File: Scripts/DCS-BIOS/lib/common/Functions.lua [hunk 1]
```diff
@@ -6,7 +6,27 @@ function ScriptPath()
...
 
+ --- @func Takes a string and checks for nil, returns 1 or 0
+ --- @param str string
+ --- @return integer
+ function Nil_state_to_int_flag(str)
+	if str ~= nil then
+		return 1
+	else
+		return 0
+	end
```
Comment (charliefoxtwo) [inline Scripts/DCS-BIOS/lib/common/Functions.lua:17]: is this used outside the apache? If not we should probably just keep it local to there for now. Though... it might have some utility in the F18 🤔 
```suggestion
	return str and 1 or 0
```

File: Scripts/DCS-BIOS/lib/modules/aircraft_modules/AJS37.lua [hunk 2]
```diff
@@ -2,7 +2,7 @@ module("AJS37", package.seeall)
 
 local Module = require("Module")
 
---- @class A_4E_C: Module
+--- @class AJS37: Module
```
Comment (charliefoxtwo) [inline Scripts/DCS-BIOS/lib/modules/aircraft_modules/AJS37.lua:5]: Good catch on this. Honestly this should probably just go in a separate PR and get merged right away.

**Review by charliefoxtwo — COMMENTED** [review]
I'm aware that some of my comments on this PR are asking for changes to code that I probably wrote years ago 🤫

File: Scripts/DCS-BIOS/lib/common/Functions.lua (hunk 1 above)
Comment (jdahlblom) [reply Scripts/DCS-BIOS/lib/common/Functions.lua:17]: fixed!

File: Scripts/DCS-BIOS/lib/modules/aircraft_modules/AH-64D.lua [hunk 3]
```diff
@@ -0,0 +1,926 @@
...
+
+local TextDisplay = require("TextDisplay")
+
+-- remove Arg# PLT 956; CPG 957
+
+--Functions
+local function parse_ku(indicator_id)
+	local ku = AH_64D.parse_indication(indicator_id)
+	if not ku then
+		return "                      " -- 22 characters
```
[the remark this answers is not in the document]
Comment (jdahlblom) [reply Scripts/DCS-BIOS/lib/modules/aircraft_modules/AH-64D.lua:18]: the string controls are defined as 22 chars long

File: Scripts/DCS-BIOS/lib/modules/aircraft_modules/AH-64D.lua [hunk 4]
```diff
@@ -0,0 +1,926 @@
...
+	end
+	return Coerce_nil_to_string(ku.Standby_text)
+end
+
+local txt_PLT_KU = ""
+local txt_CPG_KU = ""
```
[the remark this answers is not in the document]
Comment (jdahlblom) [reply Scripts/DCS-BIOS/lib/modules/aircraft_modules/AH-64D.lua:24]: removed them

File: Scripts/DCS-BIOS/lib/modules/aircraft_modules/AH-64D.lua (hunk 3 above)
[the remark this answers is not in the document]
Comment (charliefoxtwo) [reply Scripts/DCS-BIOS/lib/modules/aircraft_modules/AH-64D.lua:31]: That's correct, however `defineString` will automatically pad out any result to meet the specified length, so returning a string that is the exact correct amount of whitespace is not needed, and just creates more places where the length the string has to be defined

[the remark this answers is not in the document]
Comment (jdahlblom) [reply Scripts/DCS-BIOS/lib/modules/aircraft_modules/AH-64D.lua:31]: They should all be corrected now.

[the remark this answers is not in the document]
Comment (jdahlblom) [reply Scripts/DCS-BIOS/lib/modules/aircraft_modules/AH-64D.lua:31]: oh this is AH-54D
